packages/utils/src/ts-eslint/Linter.ts

```diff
@@ -130,12 +130,8 @@ namespace Linter {
   export type GlobalVariableOptionBase = 'off' | 'readonly' | 'writable';
   export type GlobalVariableOption = GlobalVariableOptionBase | boolean;
 
-  export interface GlobalsConfig {
-    [name: string]: GlobalVariableOption;
-  }
-  export interface EnvironmentConfig {
-    [name: string]: boolean;
-  }
```

Comment:
I believe that these should stay as interfaces to allow for global augmentation from consumers.